<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Machines</class>
 <widget class="QDialog" name="Machines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CleanApp</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>791</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion Des Machines</string>
    </attribute>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>471</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter Une Machine</string>
     </property>
     <widget class="QLineEdit" name="num_fab">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num Tel du Fabricant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fab">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Fabricant</string>
      </property>
     </widget>
     <widget class="QComboBox" name="etat_mach">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Aucun</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>En Panne</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Réparée</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="type_mach">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_mach_mod">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Supprimer Une Machine</string>
     </property>
     <widget class="QLineEdit" name="id_mach_sup">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>491</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>Afficher Les Machines</string>
     </property>
     <widget class="QLineEdit" name="chercher">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trifab">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>FAB</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tritype">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>TYPE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trietat">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>ETAT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="intervention">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>370</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gérer les interventions</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Maintenances :</string>
     </property>
    </widget>
    <zorder>groupBox</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>ajouter</zorder>
    <zorder>tableView</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>intervention</zorder>
    <zorder>label_6</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Gestion Des Produits</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Gestion Des Commandes</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
